lib/system/alloc.nim

```diff
@@ -1009,14 +901,6 @@ template instantiateForRegion(allocator: untyped) =
   proc realloc(p: pointer, newsize: Natural): pointer =
     result = realloc(allocator, p, newsize)
 
-  when false:
```

Comment:
I'd keep this (used in commented out `#sysAssert(result == countFreeMem())`)
or if you do remove it, you'd need to also remove `#sysAssert(result == countFreeMem())`